Tests/lib/vsts-task-lib/task.ts

```diff
@@ -666,9 +666,12 @@ export function matchFile(list, pattern, options): string[] {
     return mock.getResponse('match', pattern) || [];
 }
 
-export function filter(pattern, options): string[] {
-    return mock.getResponse('filter', pattern) || [];
-}    
+export function filter(pattern, options): any {
+	var filterList = mock.getResponse('filter', pattern) || [];
+	return function(pattern, i, arr) {
+		return filterList.indexOf(pattern) >= 0;
+	}
```

Comment:
seems appropriate. kind of quirky, but ultimately a better choice i think over an array of booleans